File: example-laravel/service/src/app/Http/Controllers/HealthController.php
<?php

namespace App\Http\Controllers;

use Illuminate\Http\Request;

class HealthController extends Controller
{
    public function check()
    {
        return response('OK', 200);
    }

    public function info()
    {
        return response()->json([
            'namespace' => env('SERVERLESS_NAMESPACE'),
            'container_name' => env('SERVERLESS_CONTAINER_NAME'),
            'stage' => env('SERVERLESS_STAGE'),
            'compute_type' => env('SERVERLESS_COMPUTE_TYPE'),
            'local' => env('SERVERLESS_LOCAL')
        ]);
    }
}

File: example-rails/service/app/controllers/health_controller.rb
class HealthController < ApplicationController
  def check
    render plain: 'OK'
  end

  def info
    render json: {
      namespace: ENV['SERVERLESS_NAMESPACE'],
      container_name: ENV['SERVERLESS_CONTAINER_NAME'],
      stage: ENV['SERVERLESS_STAGE'],
      compute_type: ENV['SERVERLESS_COMPUTE_TYPE'],
      local: ENV['SERVERLESS_LOCAL']
    }
  end
end
